Utils/trigger_private_build.py

```diff
@@ -0,0 +1,125 @@
+import re
+import os
+import sys
+import json
+import time
+import argparse
+import requests
+import logging
+import demisto_sdk.commands.common.tools as tools
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+INFRASTRUCTURE_FILES = ['Tests/scripts/validate_premium_packs.sh', 'Tests/scripts/validate_premium_packs.py',
+                        'Tests/scripts/validate_index.py']
+INFRASTRUCTURE_FOLDERS = ['Tests/private_build', 'Tests/Marketplace']
+
+TRIGGER_BUILD_URL = 'https://api.github.com/repos/demisto/content-private/dispatches'
+GET_DISPATCH_WORKFLOWS_URL = 'https://api.github.com/repos/demisto/content-private/actions/runs'
+WORKFLOW_HTML_URL = 'https://github.com/demisto/content-private/actions/runs'
+
+PRIVATE_REPO_WORKFLOW_ID_FILE = 'PRIVATE_REPO_WORKFLOW_ID.txt'
+
+
+def get_modified_files(branch_name):
```

Comment:
The sdk has this implemented, no need to re-implement it.
Example code
```
file_validator = ValidateManager()
file_validator.branch_name = branch_name
modified_files, added_files, _, _ = file_validator.get_changed_files_from_git()
return modified_files | added_files
```

Reply:
get_changed_files_from_git filtered only files relevant for validation(see `filter_to_relevant_files` method)
I prefer to use `get_modified_files`